### PR TITLE
[Fix bug 1129482] Add attendance filter in events advanced search.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

File: remo/events/static/events/js/events_list.js [hunk 1]
```diff
@@ -523,6 +538,16 @@ function send_query(newquery) {
         extra_q += '&campaign__name__iexact=' + initiative;
     }
 
+    if (attendance) {
+        var attendance_range = ATTENDANCE_LEVEL[attendance];
+        if (attendance_range[0]) {
+            extra_q += '&estimated_attendance__gte=' + attendance_range[0];
```
Comment (akatsoulas) [inline remo/events/static/events/js/events_list.js]: You should remove the 'equal' part from either `gte` or `lte`.

File: remo/events/static/events/js/events_list.js [hunk 2]
```diff
@@ -523,6 +538,16 @@ function send_query(newquery) {
...
+        }
+        if (attendance_range[1]) {
```
Comment (akatsoulas) [inline remo/events/static/events/js/events_list.js]: You can use an `if`  else` for the two checks.

**Comment by akatsoulas** [comment]
r+rw :beer:

File: remo/events/static/events/js/events_list.js (hunk 2 above)
Comment (johngian) [inline remo/events/static/events/js/events_list.js]: You need both checks in order to get the range. I suggest we keep this as it is.

**Pull request merged**